database/schema.sql

```diff
@@ -43,27 +49,21 @@ CREATE TABLE tag (
 );
 
 
-CREATE TABLE subscriber_tag_subscription (
-    subscription_id INT GENERATED ALWAYS AS IDENTITY,
-    subscriber_id INT NOT NULL,
-    tag_id INT NOT NULL,
-    PRIMARY KEY (subscription_id),
-    FOREIGN KEY(subscriber_id) REFERENCES subscriber(subscriber_id),
-    FOREIGN KEY(tag_id) REFERENCES tag(tag_id)
-);
 
 CREATE TABLE game (
     game_id INT GENERATED ALWAYS AS IDENTITY,
     name VARCHAR(100) NOT NULL,
     description TEXT,
     price FLOAT NOT NULL,
-    developer VARCHAR(40),
-    publisher VARCHAR(40),
+    developer_id INT,
```

Comment:
Good attention to detail by allowing `developer_id`, `publisher_id` to possibly be `NULL`

Reply:
Thanks Howard! 😄